include/NovelRT.Interop/NrtNovelRunner.h

```diff
@@ -28,10 +28,18 @@ extern "C"
     NrtResult Nrt_NovelRunner_getRenderer(NrtNovelRunnerHandle runner, NrtRenderingServiceHandle* outputService);
     NrtResult Nrt_NovelRunner_getDebugService(NrtNovelRunnerHandle runner, NrtDebugServiceHandle* outputService);
 
-    NrtResult Nrt_NovelRunner_addUpdate(NrtNovelRunnerHandle runner, void (*func)(NrtTimestamp, void*), void* context);
+    NrtResult Nrt_NovelRunner_addUpdate(NrtNovelRunnerHandle runner,
```

Comment:
where is the remove counterpart to this?

Reply:
Line 35

Reply:
damn I'm blind. Cool.